threads/tests/test_sample.py

```
@@ -0,0 +1,4 @@
+
+
+def test_sample():
+    assert 1 == 1
```

Pylint (3.3.7)

[refactor] Redundant comparison - 1 == 1

(R0124)